osbuild.spec

```diff
@@ -155,11 +159,10 @@ Contains depsolving capabilities for package managers.
 make man
 
 # SELinux
+%if %{with selinux}
 make -f /usr/share/selinux/devel/Makefile osbuild.pp
 bzip2 -9 osbuild.pp
-
-%pre selinux
```

Comment:
(Also, this one was randomly placed, I moved it in with the other selinux scripts)